docs/_config.yml

```diff
@@ -21,7 +21,7 @@ bibtex_bibfiles:
 
 # Information about where the book exists on the web
 repository:
-  url: https://github.com/RadiotherapyAI/regulatory
+  url: https://github.com/RadiotherapyAI/rai
```

Comment:
rai vs regulatory?

Reply:
Yup, that's supposed to be rai now. In fact, https://github.com/RadiotherapyAI/regulatory actually redirects to https://github.com/RadiotherapyAI/rai just to catch old links hanging around the place.

Thanks @sjswerdloff :)